README.md

```diff
@@ -40,6 +40,12 @@ of threads from any MR where your response is needed. You can configure how
 verbose the output is with options. See the `--help` option for more information
 about that.
 
+Whenever there is a new review comment that wasn't present the last time
+Reviewcheck fetched comments, you will receive a desktop notification. An
```

Comment:
Maybe mention what API is used here? Since I believe we're cool and use the libnotify API. :sunglasses:

Reply:
Nope, we just call notify-send. All the Python packages have dependencies on non-Python stuff that is rarely installed by default, and that would be too much trouble. notify-send I think we can assume most users have.

Reply:
Late reply, but I'm not sure if we should assume that. In my PR for adding a Dockerfile, I had to implement a flag that turns off notifications since the container doesn't have it. At the very least I think we could mention how a notification is sent (using notify-send(1)), since the solution of subprocess.run:ing it is a bit unorthodox. Thoughts?

Reply:
How about a new example image? I started to dislike my notification style.

Reply:
Discussed offline. Resolving.